### PR TITLE
contact-noti-2

### PR DESCRIPTION
**Pull request by Trangnguyen04798**
https://edu-redmine.sun-asterisk.vn/issues/13607

### CODE REVIEW

**Comment by Trangnguyen04798** [comment]
ready

File: resources/js/components/ContactNotification.vue [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<template>
+     <li class="m-menu__item" aria-haspopup="true">
+            <a class="m-menu__link m-menu__toggle" :href="href">
```
Comment (BunnyPi04) [inline resources/js/components/ContactNotification.vue]: indent = 4 spaces

File: resources/views/admin/layouts/head.blade.php [hunk 2]
```diff
@@ -9,9 +9,14 @@
     <link href="{{ asset('bower_components/bower/admin/demo/default/base/style.bundle.css') }}" rel="stylesheet" type="text/css" />
     <link href="{{ asset('bower_components/bower/admin/custom/css/style.css') }}" rel="stylesheet" type="text/css" />
     <link rel="stylesheet" type="text/css" href="{{ asset('bower_components/bower/admin/vendors/custom/datatables/datatables.bundle.css') }}">
+     <script>
+        window.Laravel = {!!json_encode([
```
Comment (BunnyPi04) [inline resources/views/admin/layouts/head.blade.php]: convention

**Comment by Trangnguyen04798** [comment]
ready

File: resources/js/components/ContactNotification.vue (hunk 1 above)
Comment (BunnyPi04) [reply resources/js/components/ContactNotification.vue]: comment vẫn chưa sửa à?

**Comment by Trangnguyen04798** [comment]
ready

**Pull request merged**